<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Team Yellow wiki</title>
    <link rel="shortcut icon" type="image/jpg" href="images/sand.png"/>
    <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css" rel="stylesheet">
    <link rel="stylesheet" href="styling.css">
    <link href='https://fonts.googleapis.com/css?family=Nunito|Jua|Indie Flower' rel='stylesheet'>
</head>

<body class="content_bg">
    <div class="loader_container"> <!--hourglass loader-->
        <div class="hourglass"></div>
    </div>
    <nav class="navbar"> <!--navbar-->
        <button class="menu-toggle navbar-toggle">  <!--for smaller screens-->
            <i class="fas fa-bars"></i>
        </button>

        <a href="index.html"><i class="fa-solid fa-home"></i></a>

        <div class="nav_links">
            <div class="dropdowns">
                <a href="#">Project</a>
                <div class="dropdown_contents">
                    <a href="description.html">Description </a>
                    <a href="background.html">Background</a>
                    <a href="design.html">Design</a>
                    <a href="proof_of_concept.html">Proof of Concept</a>
                    <a href="implementation.html">Implementation</a>
                    <a href="biosafety.html">Biosafety</a>
                    <a href="contributions.html">Contribution</a>
                </div>
            </div>
            <div class="dropdowns">
                <a href="#">Wet Lab</a>
                <div class="dropdown_contents">
                    <a href="engineering.html">Engineering</a>
                    <a href="materials.html">Materials</a>
                    <a href="notebook.html">Notebook</a>
                    <a href="protocols.html">Protocols</a>
                    <a href="measurement.html">Measurement</a>
                    <a href="labsafety.html">Lab Safety</a>
                    <a href="results.html">Results</a>
                </div>
            </div>
            <div class="dropdowns">
                <a href="#">Dry Lab</a>
                <div class="dropdown_contents">
                    <a href="modelling.html">Modelling</a>
                    <a href="parts.html">Parts</a>
                </div>
            </div>
            <div class="dropdowns">
                <a href="software.html">Hardware & Software</a>
            </div>
            <div class="dropdowns">
                <a href="#">Human Practices</a>
                <div class="dropdown_contents">
                    <a href="ihp.html">iHP</a>
                    <a href="outreach.html">HP</a>
                </div>
            </div>
            <div class="dropdowns">
                <a href="education.html">Education</a>
            </div>
            <div class="dropdowns">
                <a href="#">Team</a>
                <div class="dropdown_contents">
                    <a href="meet_the_team.html">Meet the Team</a>
                    <a href="attributions.html">Attributions</a>
                    <a href="medals.html">Medals</a>
                </div>
            </div>
        </div>
    </nav>

    <!--heading slide-->

    <div class="heading">
        <header><h1>Engineering</h1></header>
    </div>

    <!--content slide-->

    <div class="content-pages">

        <aside>                                 <!--table of contents-->
            <h3>Table of Contents</h3>
            <ul>
                <li><a href="#our_aim"><a>Our Aim </a></li>
                <li><a href="#the_strain_[1]">The Strain [1] </a></li>
                <li><a href="#confirmatory_tests">Confirmatory Tests</a></li>
                <li><a href="#growth_curves">Growth Curves</a></li>
                <li><a href="#plasmid_design">Plasmid Design</a></li>
                <li><a href="#wet_lab_procedures">Wet Lab Procedures</a></li>
                <li><a href="#extraction_quantification_and_mixing">Extraction, Quantification and Mixing</a></li>
            </ul>
        </aside>

        <div class="content">                    <!--content section-->
            <section class="subsections">
                
                <h2 id="our_aim"> Our Aim</h2>
                <p>
                    Our main aim is to increase the yield of γ-PGA. For that, we aim to engineer a plasmid that will produce repressor proteins suppressing the expression of the Ggt, γ-glutamyl transferase, which degrades γ-PGA [1].
                    <br><br>
                    Earlier the PgdS gene, was of interest, which is γ-PGA hydrolase, yet the team recently discovered that the resulting PgdS-deficient mutant strain did not produce γ-PGA, suggesting that the PgdS gene is essential for γ-PGA biosynthesis and its exact function is yet to be known. These studies were conducted in <i>Bacillus licheniformis</i> RK14-46 [1].
                    <br><br>
                    As discussed earlier, the disruption of the Ggt gene increased γ-PGA production due to the prevention of γ-PGA degradation. Changing the initial glucose concentration and aeration increased γ-PGA production to 39.1 ± 1.2 g/L [1].

                </p>
            </section>

            <section class="subsections">
                <h2 id="the_strain_[1]">The Strain [1]</h2>
                <p>
                    We had earlier stuck with <i>Bacillus licheniformis</i> ATCC9945A but now, the above-mentioned knockdown of the Ggt gene was performed in <i>Bacillus licheniformis</i> RK14-46. 
                    <br><br>
                    There was a sequence alignment performed between <i>Bacillus licheniformis</i> ATCC14580 and <i>Bacillus licheniformis</i> RK14-46 in the supplementary material provided by Ojima et. al., and figured out only 2 amino acids were varying. 
                    <br><br>
                    The Dry Lab team ran a BLAST between <i>Bacillus licheniformis</i> WX-02 and <i>Bacillus licheniformis</i> ATCC14580 and found a similarity of 99.89%. 
                    <br><br>
                    In a nutshell, the promoter sequences found in <i>Bacillus licheniformis</i> WX-02 are also found in <i>Bacillus licheniformis</i> ATCC14580 which is only 2 amino acids different from <i>Bacillus licheniformis</i> RK14-46 where the Ggt knockdown was successful.
                    <br><br>
                    Hence, we now prefer to stick to <i>Bacillus licheniformis</i> ATCC14580. The protocols and material document do mention <i>Bacillus licheniformis</i> ATCC9945A, yet we believe since the same species of organisms are involved, there won’t be a huge difference in the same. 

                </p>
            </section>
            <section class="subsections">
                <h2 id="confirmatory_tests">Confirmatory Tests</h2>
                <p>
                    To confirm that the outsourced bacterial culture is indeed <i>Bacillus licheniformis</i>, the team plans to conduct a series of biochemical and staining tests.                  
                    <br><br>
                    First, Gram staining will be performed to verify that the bacteria are Gram-positive rods. Hot Spore staining will help detect the presence of endospores, a characteristic feature of <i>Bacillus</i> species. 
                    <br><br>
                    The Indole test will determine whether the bacteria can break down tryptophan to produce indole. Additionally, the Methyl Red (MR) test will assess the production of stable acid end products from glucose fermentation, while the Voges-Proskauer (VP) test will confirm the ability to produce acetoin. 
                    <br><br>
                    These tests collectively provide a strong basis for identifying <i>Bacillus licheniformis</i>.
                </p>
            </section>
            <section class="subsections">
                <h2 id="growth_curves">Growth Curves</h2>
                <p>
                    The team will use controlled settings to verify growth characteristics by monitoring at various time intervals the proliferation of the outsourced <i>Bacillus licheniformis</i> culture to construct growth curves. 
                    <br><br>
                    Bacteria will be monitored at OD600 at intervals for the progression through lag, exponential, stationary, and decline phases of bacteria growth. Serial dilution and plate counting may also be utilized to obtain viable cell counts that correlate optical density with real bacterial populations. 
                    <br><br>
                    Data acquired will then be used to verify if the growth pattern conforms to the specific features predicted for <i>Bacillus licheniformis</i>, thus establishing the identity of the culture and viability for potential uses in the future. 
                    <br><br>
                </p>
            </section>
            <section class="subsections">
                <h2 id="plasmid_design">Plasmid Design</h2>
                <p>
                    pBGSC6 is an integration vector with a length of 3856 bp that confers ampicillin resistance [2]. Calcium chloride (CaCl2) competency is a method used to make <i>Bacillus licheniformis</i> cells more permeable to plasmid DNA, facilitating transformation. 
                    <br><br>
                    The process involves treating bacterial cells with an ice-cold CaCl2 solution, which helps neutralize the negative charges on both the cell membrane and DNA, allowing the plasmid to enter the cell more easily. Heat shock is then applied to create a temporary opening in the membrane, enabling DNA uptake. A PCR cycle is run to make sure the gene has been taken into the cell. 
                    <br><br>
                   
                </p>
            </section>
            <section class="subsections">
                <h2 id="wet_lab_procedures"> Wet Lab Procedures</h2>
                <p>
                    •	Transformation
                        <br><br>
                    •   Confirmation
                        <br><br>
                    •   Media Optimization
                        <br><br>
                    •	Inoculation
                        <br><br>
                    •   Incubation
                        <br><br>
                    •   Growth Curve Analysis
                        <br><br>
                    •	Fermentation
                        <br><br>
                    •   Isolation/Extraction
                        <br><br>
                    •	Quantification
                        <br><br>
                    •	Purification
                </p>
            </section>
            <section class="subsections">
                <h2 id="extraction_quantification_and_mixing">Extraction, Quantification, and Mixing</h2>
                <p>
                    The engineering method tries to optimize the transformation of <i>Bacillus licheniformis</i> to enhance gamma-PGA synthesis, as it has an application in the field of bio cement as a binding agent. The CaCl<sup>2</sup>-mediated competence approach is used to introduce the plasmid harbouring gamma-PGA synthesis genes. 
                    <br><br>
                    After transformation, growth curves and biochemical tests including gram staining, spore staining, and metabolic assays set bacterial identification and productivity. Once the gamma-PGA production is optimized, yield and quality will be tested to ensure uniformity. 
                    <br><br>
                    Successful transformation and expression of the bacterium will result in mass production of gamma-PGA, which will later be added to construction materials to enhance mechanical strength and durability.  
                    <br><br>
                    Nuclear Magnetic Resonance (NMR) spectroscopy is a powerful analytical technique for quantifying gamma-PGA in our bio cement formulation. By analyzing the distinct proton (^1H) and carbon (^13C) spectra, NMR can confirm the presence of gamma-PGA and determine its concentration with high precision. 
                    <br><br>
                    If we face issues due to technicalities and approvals, we try to resolve them by using HPLC instead. GPC is more prevalent. A gel permeation chromatography (GPC) method was developed to determine gamma-PGA volumetric yield and molecular weight directly using culture filtrates. For GPC volumetric yield measurements, a calibration curve was generated using purified gamma-PGA to relate the gamma-PGA GPC peak area and polymer weight [3]. 
                    <br><br>
                    <img src="images/hplc.png" alt="Fig. 1: GPC chromatographs of y-PGA formed by <i> B. licheniformis</i>
                    during 96 h cultivation: (a) after purification; (b) in the culture filtrate [3]" style="width: 40vw; margin-left: 13vw;">
                    <br>
                    <p style="background-color: #f0ddc0; width: 100%; text-align: center; font-family: 'Jua';">Fig. 2: GPC chromatographs of y-PGA formed by <i>B. licheniformis</i> during 96 h cultivation: (a) after purification; (b) in the culture filtrate [3]
                        </p>

                    As indicated by research, gamma-PGA should consist of about 28.6 percent of the entire solid mixture in sand mixing techniques. The values are from the ratios of polymer-to-liquid and glass-to-polymer used within biomedical cement applications. 
                    <br><br>   
                    Within our method, gamma-PGA will serve as a bonding agent, blended with sand and other aggregates to increase adhesion as well as structural strength. ultra-high-performance concrete boasts a compressive strength of approximately 130 MPa. 
                    <br><br>
                    Experimental testing will be necessary to fine-tune the ratio to optimize performance while still ensuring practical workability and durability for real-world construction applications. adjustments may be made based on field results. 
                    <br><br>
                </p>
            </section>
            <section class="subsections">       <!--references section-->
                <h2>References</h2>
                    <ol class="reference">
                        <li>Ojima, Y., Kobayashi, J., Doi, T., & Azuma, M. (2019). Knockout of pgdS and ggt gene changes poly-γ-glutamic acid production in Bacillus licheniformis RK14-46. Journal of Biotechnology, 304, 57-62.   <a class="ref" target="_blank" href="https://doi.org/10.1016/j.jbiotec.2019.08.003">https://doi.org/10.1016/j.jbiotec.2019.08.003</a></li>
                       <li><a class="ref" target="_blank" href="pBGSC6 vector map and sequence">pBGSC6 vector map and sequence</a></li>
                        <li> Birrer, G. A., Cromwick, A. M., & Gross, R. A. (1994). γ-Poly (glutamic acid) formation by Bacillus licheniformis 9945a: physiological and biochemical studies. International journal of biological macromolecules, 16(5), 265-275.
                    </ol>
            </section> 
            
        </div>
    </div>

    <footer>
        Copyright © 2024 Team Yellow
    </footer>
    
    <script src="script.js"></script>
</body>
</html>